CONTRIBUTING.md

```diff
@@ -1,54 +1,50 @@
-On the command line, you can run `make clang_format` to do the same before committing.
+To keep a consistent coding style, please install `clang-format` on your PC and run it before committing. Preferably, you should use the version we use to verify the style in our CI [(see our coding-style-check file and search for the `clangFormatVersion`)](https://github.com/jamulussoftware/jamulus/blob/master/.github/workflows/coding-style-check.yml#L20).
 
-Do not use diff/patch to send your code changes but create a Github fork of the Jamulus code and create a Pull Request when you are done.
+There are multiple ways to run clang-format:
 
-Please run a local build test. Make sure there are no errors. After opening a pull request, keep an eye on the CI checks for quality or compile issues and fix them as required.
+- Use your editor's or IDE's clang-format support to accomplish that.
+- Via make: after running `qmake Jamulus.pro`, run `make clang_format`
+- Manually: run `clang-format -i <path/to/changed/files>`
 
-If there are conflicts with jamulussoftware/jamulus showing when you go to raise the pull request, resolve those locally:
+#### Style definition
 
-```shell
-git checkout master ;# checkout your local master
-git pull upstream master --tags ;# refresh from jamulussoftware/jamulus (i.e. use your upstream remote name)
-git push --tags --force ;# update your fork's master
-git checkout - ;# back to your working branch
-git rebase master ;# replay your changes onto the latest code
-git push --force ;# update your working branch on your fork
-```
+Please see the [.clang_format file](https://github.com/jamulussoftware/jamulus/blob/master/.clang-format) in the root folder. In short:
+
+- Respect the existing code style: Tab size=4, insert spaces.
+- Add a space before and after `(` and `)`. Don't add a space between `)` and `;` or before an empty `()`.
+- Enclose all bodies of `if`, `else`, `while`, `for`, etc. in braces `{` and `}`, on separate lines.
 
 ### Supported platforms
 
 We support the following platforms and versions:
 
 - Windows 7 or higher
-- macOS <!-- Which versions? -->
-- Ubuntu Linux 18.04 or later, Debian 10 or later, most Linux flavors with recent enough Qt versions
-  <!-- Do we support BSD? I think a recent discussion was about FreeBSD? -->
-  <!-- Should we already list Android? If so, what platforms/versions? -->
+- macOS 10.10 or higher
+- Ubuntu 18.04 or later, Debian 10 or later, most Linux flavors with recent enough Qt versions
+
+Android and iOS are not officially supported, however, please ensure not to break their build.
 
 Please try to avoid breaking any of them by introducing platform-specific code. Check to see if any newly introduced Qt calls are available in the minimum supported Qt version, which is currently 5.9. Note that code _style_ in a file may be Qt 4.x, and while you should normally stick to existing style if making large-scale changes, then updating to Qt 5.9 style is encouraged.
```

Comment:
```suggestion
Please try to avoid breaking any of them by introducing platform-specific code. Check to see if any newly introduced Qt calls, parameters, properties or constants are available in the minimum supported Qt version, which is currently 5.9. Note that code _style_ in a file may be Qt 4.x, and while you should normally stick to existing style if making large-scale changes, then updating to Qt 5.9 style is encouraged.
```

cc @pgScorpio @ann0see https://github.com/jamulussoftware/jamulus/discussions/2572#discussioncomment-2488591

Reply:
Hmmm "any newly introduced ... are available in the  minimum supported Qt version" sounds a bit strange to me....

Shouldn't it be: "Do not use any new Qt functionality like components, functions, properties or property values that are introduced after the minimum supported Qt version, which is currently 5.9" ??